content/docs/guides/neon-identity.md

```diff
@@ -0,0 +1,149 @@
+---
+title: About Neon Identity
+subtitle: Automatically sync user profiles from your auth provider directly to your database
+enableTableOfContents: true
+---
+
+_<b>Q:</b> This EAP message is re-purposed from another feature. We can modify if, for example, we want to link to sign up for the EAP etc_
```

Comment:
This feature is currently available to members in our Early Access Program. To sign up, check your user profile settings in the [Neon Console](https://console.neon.tech/).